### DISCUSSION TITLE
Official Processor

### DISCUSSION THREAD

**Discussion by GretUp** [PowerJob/PowerJob#539] [General]
According to https://www.yuque.com/powerjob/en/zpp27q
we have python, script, http and file cleanup processors.
In the source code I can also see sql processor. Is there a sql official processor?
Are there some examples of how to use these official processors?

**Comment by KFCFans** [comment]
@Echo009 It's time to show your English skills

**Comment by KFCFans** [comment]
here is the chines doc, maybe you can use translation software to have a look

https://www.yuque.com/powerjob/guidence/official_processor#9LrIO

**Comment by GretUp** [comment]
OK, thanks. Will have a look.
